locales/en.json

```diff
@@ -71,28 +71,28 @@
 	"Header": "Header",
 	"Unordered List": "Unordered List",
 	"Ordered List": "Ordered List",
-	"Todo List": "Todo List",
+	"Todo List": "Checklist",
 	"Blockquote": "Blockquote",
 	"Bold font": "Bold font",
 	"Italics font": "Italics font",
 	"Strikethrough": "Strikethrough",
 	"Inserted text": "Inserted text",
-	"Marked text": "Marked text",
+	"Marked text": "Highlighted text",
```

Comment:
This came from the markdown engine we're using `markdown-it`.
You could see they use "Marked text" in their demo https://markdown-it.github.io/.
And the tag of the marked text is `<mark>` which pretty make sense.